R/readSDMX.R

```diff
@@ -151,7 +151,8 @@ readSDMX <- function(file = NULL, isURL = TRUE,
                      validate = FALSE, verbose = TRUE) {
   
   #set option for SDMX compliance validation
-  .rsdmx.options$validate = validate
+  .rsdmx.options$validate <- validate
+  .rsdmx.options$followlocation <- TRUE
```

Comment:
It allows curl to follow redirections (3xx status codes) instead of otherwise rejecting them as failed requests (status codes 4xx and up).